src/components/OrgFile/components/AttributedString/index.js

```diff
@@ -11,26 +13,46 @@ import classNames from 'classnames';
 export default ({ parts, subPartDataAndHandlers }) => {
   let className;
 
+  let location = useLocation();
+
+  const renderLink = (part) => {
+    const id = part.get('id');
+    const uri = part.getIn(['contents', 'uri']);
+    const title = part.getIn(['contents', 'title']) || uri;
+    if (uri.startsWith("file:")) {
+      const target = uri.substr(5);
+      const isRelativeOrgFileLink =
+        !target.startsWith("/") &&
+        !target.startsWith("~") &&
+        !target.startsWith("../") &&
```

Comment:
Actually, this is also a relative link. But OK, it's not trivial to fix it against "sync directory break-out".

Reply:
Not trivial, but not too hard either ;-)  I've done it.